README.md

```diff
@@ -1,46 +1,72 @@
-# Event Publisher Proxy
+# Eventing Publisher Proxy
 
 ## Overview
 
-The Event Publisher Proxy receives legacy and Cloud Event publishing requests from the cluster workloads (microservice or Serverless functions) and redirects them to the Enterprise Messaging Service Cloud Event Gateway. It also fetches a list of subscriptions for a connected application.
+The Eventing Publisher Proxy receives legacy and Cloud Event publishing requests from the cluster workloads (microservice or Serverless functions) and redirects them to the Enterprise Messaging Service Cloud Event Gateway. It also fetches a list of subscriptions for a connected application.
 
 ## Prerequisites
 
-- Go modules
-- [ko](https://github.com/google/ko)
-- lint
-  - Install lint on the local environment:
-```bash
-curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin
-```    
+- [Go](https://go.dev/)
+- [Docker](https://www.docker.com/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kustomize](https://kustomize.io/)
+- Access to Kubernetes cluster ([k3d](https://k3d.io/) / k8s)  
 
 ## Development
-- check code quality before a commit:
-```bash
-$ make code-check
-```
 
 ### Build
 
 ```bash
-$ go mod vendor
+make build
 ```
 
-### Test
+### Run tests
 
-```bash
-$ make test-local
+Run the unit and integration tests:
+
+```sh
+make generate-and-test
+```
+
+### Linting
+
+1. Fix common lint issues:
+
+   ```sh
+   make imports
+   make fmt
+   make lint
+   ```
+
+### Build container images
+
+Build and push your image to the location specified by `IMG`:
+
+```sh
+make docker-build docker-push IMG=<container-registry>/eventing-publisher-proxy:<tag> # If using docker, <container-registry> is your username.
+```
+
+> **NOTE**: For MacBook M1 devices, run:
```

Comment:
```suggestion
For MacBook M1 devices, run:
```
Either put the code block into the Note box as well, or simply drop the note box. My suggestion is for the simpler solution :D